css/src/components/popover.scss

```diff
@@ -0,0 +1,36 @@
+/* stylelint-disable scss/at-extend-no-missing-placeholder */
+.popover {
+	display: inline-block;
+	position: relative;
+
+	.popover-summary {
+		@extend .button;
+
+		min-height: unset;
+	}
+
+	.popover-content {
+		position: absolute;
+		min-width: 150px;
+		margin-top: $spacer-3;
+		padding: $spacer-6;
+		border: 1px solid $border;
+		border-radius: $border-radius;
+		background-color: $body-background;
+		box-shadow: $box-shadow-heavy;
+		z-index: $zindex-popover;
+
+		&.popover-content-align-left {
+			left: 0;
+		}
```

Comment:
is this required? Thinking left alignment happens by default even if left:0 is not set.

Reply:
Oh, you're right, forgot to remove it. Initial idea was to keep it in case of the need to overwrite another rule.

Reply:
removed that one